libs/SmartSync/SmartSync/Classes/Util/SFSyncTarget.m

```diff
@@ -1,61 +0,0 @@
-/*
```

Comment:
Moved all target related code to a target sub-directory (on android we did the same thing).